content/version/1/4/code-of-conduct.md

```diff
@@ -28,34 +28,35 @@ include:
 Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or
-  advances
```

Comment:
This is bizarre. If something like this happened in a workplace, there wouldn't be any question, the person would be fired. Why should people working on open source projects be more accommodating to unwelcome sexual behavior than any other workplace?